### ISSUE TITLE
Discord or some other community hub

### ISSUE THREAD

**Issue by lUNuXl** [rjaros/kvision#296]
Hello,

I've noticed that this framework doesn't have any community hub, it would be really handy to have one since it's not a popular framework.

The slack channel doesn't really count as it's impossible to login to that instance. I know that you can fill in some form to get an account there but from what I can tell it's not automated.

A discord server would be a really handy solution - anyone having difficulties or anyone who just stumbled upon this project could just join the server and start getting answers and help from the others.

**Comment by HallofFamer** [comment]
Yeah would be nice if KVision has a dedicated discord community or discussion board, though I wonder how many people will be there.
